NameGame/NameGame/UI/Menu/MenuWorkflow.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  MenuWorkflow.swift
+//  NameGame
+//
+//  Created by Tammy Le on 6/23/17.
+//  Copyright © 2017 WillowTree Apps. All rights reserved.
+//
+
+import Foundation
+import ReSwift
+
+protocol CoordinatorHandler {
+    
+    func createUserGridCoordinator()
+    func createGameCoordinator()
+}
+
+final class MenuWorkflow {
+    
+    let store: Store<AppState>
+    private(set) var userRecordService: UserRecordService!
+    var coordinatorHandler: CoordinatorHandler!
```

Comment:
Also, if we rename the protocol; rename this property to `presentationEventHandler`.